template/v1/Dockerfile

```diff
@@ -60,6 +66,14 @@ RUN micromamba install -y --name base --file /tmp/$ENV_IN_FILENAME && \
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
 RUN sudo ln -s $(which python3) /usr/bin/python
 
+# Merge in OS directory tree contents.
```

Comment:
Why are we moving, is it intentional to be done by MAMBA_USER instead of ROOT ?

Reply:
The `dirs/` directory contains the extensions, settings file needed to configure CodeEditor - so this needs to happen before `TODO - Configure CodeEditor ` line below.

The change is user is not intentional though, thanks - will update